cargo-near/src/commands/new/new-project-template/.github/workflows/undeploy-staging.yml

```diff
@@ -1,6 +1,7 @@
 name: Undeploy staging
 on:
   pull_request:
+    # TODO: GitReference::RevPlusRef
     types: [closed]
```

Comment:
according to following documentation https://docs.github.com/en/webhooks/webhook-events-and-payloads?actionType=closed#pull_request

![image](https://github.com/user-attachments/assets/3c97f9a4-a4db-45aa-903a-ec2892b5c754)

Reply:
checked that `undeploy-staging.yml` runs on both prs merged/closed